staging/src/k8s.io/pod-security-admission/test/fixtures_test.go

```diff
@@ -46,7 +46,7 @@ func TestFixtures(t *testing.T) {
 
 	defaultChecks := policy.DefaultChecks()
 
-	const newestMinorVersionToTest = 23
+	const newestMinorVersionToTest = 24
```

Comment:
I don't think there is anything 1.24-specific in the policies, so can revert this to 23

Reply:
Done.